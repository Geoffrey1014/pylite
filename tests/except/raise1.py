def main():
    raise Exception  # throw class

print "Start"
main()
print "End"
