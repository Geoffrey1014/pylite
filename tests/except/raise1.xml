lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    load_const c0
    store_global c1
    load_const c2
    print
    load_global c1
    call 0
    pop
    load_const c3
    print
    load_const c4
    return
  const_vector
    lambda
      name main
      param_count 0
      local_count 0
      label_count 0
      insn_vector
        load_global c0
        raise
        load_const c1
        return
      const_vector
        Exception
        None
      guard_vector
      line_number_map 2 2 1 1
    main
    "Start"
    "End"
    None
  guard_vector
  line_number_map 1 1 4 4 5 5 5 6 6 1 1
