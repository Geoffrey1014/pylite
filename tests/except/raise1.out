Start
Exception:
  At raise1.py, line 2, in main()
  At raise1.py, line 5, in __init__()
